app-backend/database/src/main/scala/com/azavea/rf/database/tables/Exports.scala

```diff
@@ -183,25 +183,27 @@ object Exports extends TableQuery(tag => new Exports(tag)) with LazyLogging {
 
     eo.flatMap({ exportOptions =>
 
-        val outDef = OutputDefinition(
-          crs = exportOptions.getCrs,
-          rasterSize = exportOptions.rasterSize,
-          render = Some(exportOptions.render),
-          crop = exportOptions.crop,
-          stitch = exportOptions.stitch,
-          source = exportOptions.source,
-          dropboxCredential = user.dropboxCredential
-        )
+      val outDef = OutputDefinition(
+        crs = exportOptions.getCrs,
+        rasterSize = exportOptions.rasterSize,
+        render = Some(exportOptions.render),
```

Comment:
But "casted" to `Option`, not `Some`. Having it appear as `Option` injects `|@|` correctly.